core/bugnote_api.php

```diff
@@ -192,25 +194,27 @@ function bugnote_add( $p_bug_id, $p_bugnote_text, $p_time_tracking = '0:00', $p_
 
 	antispam_check();
 
+	$t_bugnote_text = $p_bugnote_text;
+
```

Comment:
Assignment not needed if $p_bugnote_text is used (again) in following two is_blank() checks.